### PR TITLE
Markov Proc

### PR DESCRIPTION
**Pull request by Shadowmech88**
Adds a `markov_chain()` proc which takes a string and returns a string containing a generated markov chain using the words in the input string.

I don't know if it could be different or more optimized or what, I just attempted to 1:1 port the first example I found on Google.

I plan to maybe use this in a later feature but I thought it'd be good to add this by itself first.

### CODE REVIEW

**Comment by PJB3005** [comment]
I should dump MoMMI's Markov database into this and see how hilariously slow it is.

**Comment by Iamgoofball** [comment]
I'm not seeing where to pre-feed it

**Comment by PJB3005** [comment]
You don't, it has no global state.

**Comment by Iamgoofball** [comment]
Gross, how am I going to feed in My Immortal now?

**Comment by Shadowmech88** [comment]
By passing the whole thing in as an argument in string form.

**Comment by Iamgoofball** [comment]
Jesus Christ

File: code/__HELPERS/markov.dm [hunk 1]
```diff
@@ -0,0 +1,59 @@
+/proc/markov_chain(var/text, var/order = 4, var/length = 250)
+	if(!text || order < 0 || order > 20 || length < 1 || length > 25000)
```
Comment (ComicIronic) [inline code/__HELPERS/markov.dm]: Make the length maximum a define.

Comment (Shadowmech88) [reply code/__HELPERS/markov.dm]: Done.

**Pull request merged**